tajo-cli/src/main/java/org/apache/tajo/cli/tsql/commands/ConnectDatabaseCommand.java

```diff
@@ -63,7 +63,7 @@ public void invoke(String[] cmd) throws Exception {
 
   @Override
   public String getUsage() {
-    return "";
+    return "[database_name]";
```

Comment:
How about use ```DBNAME``` instead of ```database_name```?  TSQL already uses ```DBNAME``` to print help descriptions with ```\help``` parameter.